### PR TITLE
Ft build workfile clear scene prop

### PR DESCRIPTION
**Pull request by kaamaurice**
## Changelog Description
Added `clear scene` property for build workfile operator.

### CODE REVIEW

File: openpype/hosts/blender/api/ops.py [hunk 1]
```diff
@@ -1016,27 +1020,26 @@ def __init__(self):
...
-            bpy.data.objects.remove(obj)
-        for collection in set(bpy.data.collections):
-            bpy.data.collections.remove(collection)
-        # purgne unused datablock
-        while bpy.data.orphans_purge(do_local_ids=False, do_recursive=True):
-            pass
-        # clear all libraries
-        for library in list(bpy.data.libraries):
-            bpy.data.libraries.remove(library)
+    def _build_first_workfile(self, clear_scene, save_as):
```
Comment (Tilix4) [inline openpype/hosts/blender/api/ops.py]: Could you document this please? Hints and docstring

**Review by Tilix4 — CHANGES_REQUESTED** [review]
Thanks!

File: openpype/hosts/blender/api/ops.py (hunk 1 above)
Comment (kaamaurice) [reply openpype/hosts/blender/api/ops.py]: yep

Comment (Tilix4) [reply openpype/hosts/blender/api/ops.py]: you forgot hints :)

Comment (kaamaurice) [reply openpype/hosts/blender/api/ops.py]: oups

Comment (kaamaurice) [reply openpype/hosts/blender/api/ops.py]: done :)

**Review by Tilix4 — APPROVED** [review]
LGTM!

**Pull request merged**